src/types/parameter/parameter_enums.enums.yaml

```diff
@@ -55,3 +61,12 @@ enums:
       - name: Dropped
         value: 4
         description: The operation could not be performed because it was dropped from a full queue.
+  - name: Parameter_Validation_Status
+    description: This status enumeration provides information on the result of a parameter validation.
+    literals:
+      - name: Valid
+        value: 0
+        description: Parameter was validated.
+      - name: Invalid
+        value: 1
+        description: Parameter was found to be invalid.
```

Comment:
Doesn't this more like validate a set of parameters together. The descriptions sound like this is validating just a single parameter. Maybe reword so it sounds like we are validating a parameter set.

Reply:
Reworded in [87b53e2](https://github.com/lasp/adamant/pull/64/commits/87b53e25918b9ba58654c3b49c2e6f54074784cb).